src/main/resources/templates/email/programme-updated-week-0/v1.0.0.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">Programme checklist: 8 weeks to go</th:block>
```

Comment:
These (0-, 1-, and 4- week notification templates) are just placeholders at this point (all copies of the 8-week notification)